src/store-api/src/region_request.rs

```diff
@@ -661,23 +665,27 @@ impl From<v1::ChangeColumnType> for ChangeColumnType {
 #[derive(Debug, Eq, PartialEq, Clone, Serialize, Deserialize)]
 pub enum ChangeOption {
     TTL(Duration),
+    Twsc(String, String),
 }
 
 impl TryFrom<&ChangeTableOption> for ChangeOption {
     type Error = MetadataError;
 
     fn try_from(value: &ChangeTableOption) -> std::result::Result<Self, Self::Error> {
         let ChangeTableOption { key, value } = value;
-        if key == TTL_KEY {
-            let ttl = if value.is_empty() {
-                Duration::from_secs(0)
-            } else {
-                humantime::parse_duration(value)
-                    .map_err(|_| InvalidRegionOptionChangeRequestSnafu { key, value }.build())?
-            };
-            Ok(Self::TTL(ttl))
-        } else {
-            InvalidRegionOptionChangeRequestSnafu { key, value }.fail()
+
+        match key.as_str() {
+            TTL_KEY => Ok(Self::TTL(humantime::parse_duration(value).map_err(
```

Comment:
The old logic is when the value is empty, the TTL value is `Duration::from_secs(0)`. Looks like it changes the logic here.

Reply:
yes that is one of the reason sqlness test failed, fixed in update.